### DISCUSSION TITLE
The live software  "直播姬(Zhi Bo Ji)"  developed by China  Bilibili Technology Co., Ltd. uses the source code of OBS-project

### DISCUSSION THREAD

**Discussion by evilevenmore** [obsproject/obs-studio#4937] [Development Questions]
At first,they didn't follow the rules of GPL.
Then the problem was discovered by an user and posted on social media, so in 2016 they(Bilibili) uploaded the source code to github
Now, their live broadcast software called "直播姬(Zhi Bo Ji)" is still being updated, but the github repository has stopped updating the source code for five years.
Is this allowed in the GPL agreement?

https://github.com/Bilibili/biliobs

**Comment by notr1ch** [comment]
Technically they only have to provide the source code on a request by someone they have distributed the software to. Maintaining an up to date GitHub repository is an easy way to fulfill GPL requirements but not required.
